sixtyfps_runtime/rendering_backends/gl/graphics_window.rs

```diff
@@ -218,15 +124,21 @@ impl GraphicsWindow {
             }
         };
 
-        let backend = self.window_factory.as_ref()(window_builder);
+        let backend = crate::GLRenderer::new(
```

Comment:
Is `backend` the right name here?

Do we even need GLRenderer at all

Reply:
We do not, that’s right. I wanted to continue to with that but the patch become a bit large :-)